frontend/js/components/animatecounters.js

```diff
@@ -0,0 +1,30 @@
+export default function () {
+  let $counter = $(".counter");
```

Comment:
It's better to use a prefix with the selector. Such as `$(".mixed-content-module .counter")`. Thus we won't affect other parts on the page.